tsdb/store.go

```diff
@@ -224,6 +226,11 @@ func (s *Store) loadShards() error {
 			continue
 		}
 
+		if s.EngineOptions.DatabaseFilter != nil && !s.EngineOptions.DatabaseFilter(db.Name()) {
+			log.Info("Skipping database dir", zap.String("name", db.Name()), zap.String("reason", "failed database filter"))
```

Comment:
Maybe `logger.Database`? And I guess we're skipping a database, regardless of if it's kept in a directory or no 😄

Reply:
Yeah, in this case it should be `database`; I copied it from the log line above that skips because it _isn't_ a directory.